agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1989,6 +1990,50 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
     // This is something that cannot be done at this moment because the MediaType field
     // must be computed with information obtained through NL80211_CMD_GET_WIPHY command of DWPAL,
     // which is currently not supported. See "Send standard NL80211 commands using DWPAL #782"
+    // For now, fill from radio data with dummy values based on information we have.
+    for (const auto soc : slaves_sockets) {
```

Comment:
```suggestion
    for (const auto &soc : slaves_sockets) {
```

Reply:
Basically it is a pointer (shared), but you are right, a shared pointer size is 16 and is greater than 4.
fixed.